src/components/Modal/SettingsModal.tsx

```diff
@@ -78,13 +84,18 @@ export const SettingsModal: React.FC<{
     }
   ]
 
+  const _closeModal = () => {
+    closeModal()
+    setSelectedViewId(ViewIds.main)
```

Comment:
Ah so this is what you mentioned the other day. A better fix would be to configure the view id to change to on a per screen basis <https://github.com/pooltogether/pooltogether-react-components/blob/e949da9a958a0fb4d03af80c6524530cbe05e31c/src/components/Modal/ModalWithViewState.tsx#L8>